backstop_data/engine_scripts/puppet/page-objects/crm-page.js

```diff
@@ -98,6 +99,22 @@ module.exports = class CrmPage {
     await Promise.all([
       this.engine.click(selector),
       this.engine.waitForNavigation()
-    ]);      
+    ]);
+  }
+
+  /**
+   * Checks if there are no price fields present on the page or not
+   * @param {String} type - the title of the message to be displayed
```

Comment:
Now because you added a param, a line break is needed

```js
/**
 * Checks if there are no price fields present on the page or not
 *
 * @param {String} type - the title of the message to be displayed
 */
```

Reply:
Done!